toolchain/sem_ir/type.h

```diff
@@ -209,7 +209,7 @@ class TypeStore : public Yaml::Printable<TypeStore> {
 
 // If `type_id` is a `PatternType`, returns its scrutinee type; otherwise,
```

Comment:
Let's update this comment now that the implementation CHECKs?

Reply:
Whoops, good catch.